### PR TITLE
Design/ Start page

### PR DESCRIPTION
**Pull request by AljoschaNy**

### CODE REVIEW

File: frontend/src/pages/StartPage.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
+.start-main-container {
+    background: var(--color-1);
```
Comment (mpagels) [inline frontend/src/pages/StartPage.css:2]: Highly recommend not using names like "--color-1". Try to use something like more descriptive names like "background-main-color" or "background-secondary-color" So you will always know which color it is. "--color-1" could be all or nothing.

File: frontend/src/pages/StartPage.tsx [hunk 2]
```diff
@@ -1,13 +1,33 @@
...
+            <main className={"main-start"}>
+                <section className={"banner"}>
+                    <div className={"banner-video-section"}>
+                        <video className={"banner-video"} src={"/banner-video.mp4"} autoPlay loop muted />
+                        <div className={"video-overlay"}></div>
+                    </div>
+                    <div className={"banner-text"}>
+                        <h2>Deine Fitness.</h2>
+                        <h2>Deine Kontrolle.</h2>
+                        <h2>Dein Erfolg.</h2>
```
Comment (mpagels) [inline frontend/src/pages/StartPage.tsx:22]: Always use descending headlines.
Do you really need headlines here? Looks more like a list.

**Review by mpagels — CHANGES_REQUESTED** [review]
A pull request description would be nice. It helps the reviewer on what to focus and what happened here.

Found a headline issue. And put some small recommendation in there. Pls review.

File: frontend/src/pages/StartPage.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    height: 65px;
+    padding: 0 30px;
+    background: var(--color-1);
+    display: flex;
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 200;
+    justify-content: space-between;
+    align-items: center;
```
[the remark this answers is not in the document]
Comment (AljoschaNy) [reply frontend/src/pages/StartPage.css:16]: Ah ja stimmt

File: frontend/src/pages/StartPage.tsx (hunk 2 above)
Comment (AljoschaNy) [reply frontend/src/pages/StartPage.tsx:22]: Stimmt wurde geändert

**Comment by AljoschaNy** [comment]
@mpagels I implemented the changes

**Review by mpagels — APPROVED** [review]

**Pull request merged**